apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -429,6 +447,27 @@ export const PoolPoster = ({
                 />
               </div>
             </TableOfContents.Section>
+            {showSpecialNote && (
+              <TableOfContents.Section id={sections.specialNote.id}>
+                <div>
+                  <Alert.Root
+                    type="info"
+                    data-h2-shadow="base(none)"
+                    data-h2-margin="base(0, -x1, 0, -x1)"
```

Comment:
I think we could probably remove the margin here. Looks a little odd when it extends past the content bounds.

Reply:
Opps missed that, thanks! Fixed by [c8f964f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8006/commits/c8f964f6e9e379116e89a87c15145faeea45a688)